ReviewBot.py

```diff
@@ -765,6 +765,15 @@ def comment_write(self, state='done', result=None, project=None, package=None,
 
         self.comment_handler_remove()
```

Comment:
@DimStar77 as we learned this week my logic code can use a reviewer. So if you may...